packages/sensors/lib/sensors.dart

```diff
@@ -40,16 +62,42 @@ class GyroscopeEvent {
   String toString() => '[GyroscopeEvent (x: $x, y: $y, z: $z)]';
 }
 
+class LinearAccelerometerEvent {
```

Comment:
Should it be renamed to UserAccelerationEvent following the iOS naming?
I think that is somehow easier to understand.

Reply:
I love that, no idea how that didn't occur to me!